File: src/test/java/ccpl/lib/BundleTest.java
package ccpl.lib;

import static junit.framework.TestCase.assertTrue;

import org.junit.Test;

public class BundleTest {

  @Test
  public void getAsBoolean() {
    Bundle tempBundle = new Bundle();

    tempBundle.add("string", "true");
    tempBundle.add("boolean", true);

    assertTrue(tempBundle.getAsBoolean("string"));
    assertTrue(tempBundle.getAsBoolean("boolean"));
  }
}

File: src/test/kotlin/ccpl/numberline/BundleTest.kt
package ccpl.numberline

import ccpl.lib.Bundle
import org.junit.Test

import org.junit.Assert.*

class BundleTest {
  @Test
  fun getAsBoolean() {
    val tempBundle = Bundle()

    tempBundle.add("string", "true")
    tempBundle.add("boolean", true)

    assertTrue(tempBundle.getAsBoolean("string"))
    assertTrue(tempBundle.getAsBoolean("boolean"))
  }

}